assets/javascripts/admin_user.js

```diff
@@ -37,4 +37,23 @@ function setup_admin_user() {
             }
         });
     });
+
+    window.deleteUser = function(id){
+        if (confirm("Are you sure you want to delete this user?")) {
+            $.ajax({
+                url: '/api/v1/user/' + id,
+                method: 'DELETE',
+                dataType: 'json',
+                success: function() {
+                  addFlash('info', 'The user was deleted successfully.');
+                  var row = window.admin_user_table.row($("#user_" + id));
+                  row.remove().draw();
+                },
+                error: function(xhr, ajaxOptions, thrownError) {
+                  var error_message = xhr.responseJSON.error;
```

Comment:
If the server wasn't reachable at all there won't be any response which should be handled.